### PR TITLE
improve: hard coded zoom to app.slug

### LINKED ISSUE #11856: chore: use app.slug not hard coded "zoom"
> I think we can improve on this PR by instead of hard coding "zoom" we can use the app.slug directly.

lets do a follow up PR

_Originally posted by @PeerRich in https://github.com/calcom/cal.com/issues/11795#issuecomment-1759551870_

### PR DESCRIPTION
**Pull request by MeenuyD**
## What does this PR do?
Use app.slug dynamically instead of hardcoding 'zoom' for improved flexibility.

Fixes #11856

### CODE REVIEW

File: packages/app-store/zoomvideo/lib/VideoApiAdapter.ts [hunk 1]
```diff
@@ -91,7 +91,8 @@ const zoomAuth = (credential: CredentialPayload) => {
             grant_type: "refresh_token",
           }),
         }),
-      "zoom",
+
+      app.slug,
```
Comment (PeerRich) [inline packages/app-store/zoomvideo/lib/VideoApiAdapter.ts:95]:
```suggestion
      app.slug,
```

**Review by PeerRich — DISMISSED** [review]

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Thanks for your contribution @MeenuyD.

`app` does not exist in the `VideoApiAdapter` so this throws an error. We should find another way to dynamically get the app slug. We should also do this for other apps as well.

**Comment by MeenuyD** [comment]
Ok I will try

**Comment by MeenuyD** [comment]
Can you tell me how can we do

**Comment by Siddharth-2382** [comment]
> Can you tell me how can we do

add `import metadata from "zoomvideo/_metadata";` at the top.
replace `app.slug` with `metadata.slug`

**Comment by PeerRich** [comment]
already fixed

**Pull request closed without merging**